packages/dd-trace/src/util.js

```diff
@@ -20,8 +20,20 @@ function isError (value) {
   return false
 }
 
+function parseRules (rules) {
+  if (Array.isArray(rules)) return rules
+  if (typeof rules !== 'string') return []
```

Comment:
Since env vars are always strings, this could be merged with the below statement as `JSON.parse(rules || '[]')`.